src/lib/y2network/widgets/destination.rb

```diff
@@ -24,8 +24,11 @@ def contents
       end
 
       def help
-        # TODO: original also does not have help, so write new one
-        ""
+        _(
+          "<p><b>Default Route</b> matches all destination for given IP " \
+            "version as long as no specific route matches. <b>Destination</b>" \
+            "specifies IP address with prefix for which route applies.</p>"
```

Comment:
what about *specifies the IP address (in CIDR format) for which the route applies* ? Using *prefix* instead of *CIDR*  is fine for me. Just not the *the*.